diff -ruN .gitignore .gitignore
--- .gitignore	1970-01-01 02:00:00.000000000 +0200
+++ .gitignore	2023-07-03 17:26:58.942364078 +0300
@@ -0,0 +1,2 @@
+*
+!.gitignore
diff -ruN src/contracts/StakedTokenV3.sol src/contracts/StakedTokenV3.sol
--- src/contracts/StakedTokenV3.sol	2023-07-26 19:06:56.827296427 +0300
+++ src/contracts/StakedTokenV3.sol	2023-07-26 11:49:16.510757673 +0300
@@ -580,11 +580,14 @@
     address user,
     GovernancePowerType delegationType
   ) public view override returns (uint256) {
-    return
-      (super.getPowerCurrent(user, delegationType) * EXCHANGE_RATE_UNIT) /
-      getExchangeRate();
+      return mul_div_munged(super.getPowerCurrent(user, delegationType), getExchangeRate());
   }
 
+  function mul_div_munged(uint256 x, uint256 denominator) public view returns (uint256) {
+      return (x*EXCHANGE_RATE_UNIT)/denominator;
+  }
+
+  
   function _setDelegationState(
     address user,
     DelegationState memory delegationState
